<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPresence</class>
 <widget class="QDialog" name="DlgPresence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Presence</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>82</width>
     <height>158</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbOnline">
      <property name="text">
       <string>Online</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbOffline">
      <property name="text">
       <string>Offline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbAway">
      <property name="text">
       <string>Away</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbBusy">
      <property name="text">
       <string>Bysy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbOk">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
